lib/iris/experimental/um.py

```diff
@@ -687,6 +687,23 @@ def normalise(values, kind):
             sector_size = self._WORDS_PER_SECTOR * self._word_size
             for field in self.fields:
                 data = field.get_data()
+
+                if hasattr(field, "lbpack"):
+                    lbpack = field.lbpack
+                    packing = lbpack % 10
+                    compression = (lbpack // 10) % 10
+                    number_format = (lbpack // 1000) % 10
+                    unsupported = False
+                    if packing not in (0, 1):  # Unpacked/WGDOS
+                        unsupported = True
+                    if compression != 0:  # Uncompressed
+                        unsupported = True
+                    if number_format not in (0, 2, 3):  # Native/Cray/IEEE
+                        unsupported = True
+                    if unsupported:
```

Comment:
This would be more naturally expressed as:
```python
if packing not in (0, 1) or compression != 0 or number_format not in (0, 2, 3):
```
That would also allow for [short-circuiting](http://en.wikipedia.org/wiki/Short-circuit_evaluation).